app/src/main/java/com/duckduckgo/app/downloads/DownloadsAdapter.kt

```diff
@@ -26,23 +26,28 @@ import com.duckduckgo.app.browser.R
 import com.duckduckgo.app.browser.R.layout
 import com.duckduckgo.app.browser.databinding.ViewItemDownloadsEmptyBinding
 import com.duckduckgo.app.browser.databinding.ViewItemDownloadsHeaderBinding
+import com.duckduckgo.app.browser.databinding.ViewItemDownloadsNotifyMeBinding
 import com.duckduckgo.app.downloads.DownloadViewItem.Empty
 import com.duckduckgo.app.downloads.DownloadViewItem.Header
 import com.duckduckgo.app.downloads.DownloadViewItem.Item
+import com.duckduckgo.app.downloads.DownloadViewItem.NotifyMe
 import com.duckduckgo.app.global.formatters.data.DataSizeFormatter
 import com.duckduckgo.downloads.store.DownloadStatus.FINISHED
 import com.duckduckgo.mobile.android.databinding.RowTwoLineItemBinding
 import com.duckduckgo.mobile.android.ui.menu.PopupMenu
+import com.duckduckgo.mobile.android.ui.notifyme.NotifyMeListener
 import com.duckduckgo.mobile.android.ui.view.gone
 import com.duckduckgo.mobile.android.ui.view.show
 import javax.inject.Inject
+import kotlinx.android.synthetic.main.view_item_downloads_notify_me.view.*
```

Comment:
We don't use synthetics anymore, see https://app.asana.com/0/414730916066338/1198731892860153/f